### ISSUE TITLE
capture help doesn't show all options

### ISSUE THREAD

**Issue by dmcassel** [marklogic-community/roxy#304]
```
$ ./ml capture -h
Usage: ml {env} capture --modules-db=[name of modules database]
  Captures the source and REST API configuration for an existing
  Application Builder-based application.

modules-db: (required)
  The modules database of the App Builder application.
```

The capture command provides additional options, like --full-ml-config, --ml-config, and --app-builder. Update the help command to show those options and what they do.

**Comment by DavidSewell** [comment]
Hello, a new Roxy user here. I was just about to file an issue ticket on this exact same thing. I think it would be very helpful to new users to have the complete documentation for "ml capture" returned in the help text -- I knew that roxy could return full configuration information only from googling "roxy capture" and finding a couple of blog posts about using --full-ml-config.

**Comment by grtjn** [comment]
Yeah, the suggested help text in here is most recent, but even that might be little outdated, since I think there was some work done on capturing security stuff as well..
